src/applications/claims-status/utils/helpers.js

```diff
@@ -146,6 +146,18 @@ export function displayFileSize(size) {
   return `${Math.round(mbSize)}MB`;
 }
 
+export function groupClaimsByDocsNeeded(list) {
```

Comment:
You don't have to do it this way, but another way to do this would be something like this:
```javascript
export function groupClaimsByDocsNeeded(list) {
  const hasOpenRequests = claim => claim.attributes.documentsNeeded;

  const claimsWithOpenRequests = list.filter(hasOpenRequests);

  const claimsWithoutOpenRequests = list.filter(
    claim => !hasOpenRequests(claim),
  );

  return claimsWithOpenRequests.concat(claimsWithoutOpenRequests);
}
```